index.html

```diff
@@ -0,0 +1,116 @@
+  <button type="button" id="close-menu" class="close-menu">&times;</button>
+  <a href="index.html"><img class="menu-logo" src="./src/icons/logo.png" alt="logo"></a>
+</div>
+  <div class="m-menu-container">
+    <a class="lato black menu-btn" href="about.html">About</a>
+    <a class="lato black menu-btn" href="#" onclick="closeNav()">Program</a>
+    <a class="lato black menu-btn" href="#" onclick="closeNav()">Join</a>
+    <a class="lato black menu-btn" href="#" onclick="closeNav()">Sponsor</a>
+    <a class="lato black menu-btn" href="#" onclick="closeNav()">News</a>
+    <a class="lato orange menu-btn" href="#" onclick="closeNav()">RE Campaing</a>
+  </div>
+</div>
+    </header>
+    <section class="start-section">
+<p class="start-title-p orange lato">"Hello! Sharing world"</p>
+<h1 class="start-title orange coco">RENEWABLE ENERGY GLOBAL SUMMIT 2022</h1>
+<div class="start-p-container">
+<p class="start-descr black lato">It is our pleasure to invite all scientists, academicians, young researchers, business delegates
+   and students from all over the world to attend the RE Global Summit 2022 this April in Canada.</p>
+  </div>
+   <p class="start-date black coco">2022.04.14(THU) ~ 15(FRI)</p>
+   <p class="start-place black lato">@ University of Toronto - Canada</p>
+</section>
+<section class="main-section">
+  <h2 class="main-title gray lato">Main Program</h2>
+  <hr class="main-hr">
+    <ul class="main-ul">
+      <li class="main-li">
+        <img src="src/icons/lecture.webp" alt="Lecture Icon" class="icon">
+        <h3 class="orange main-li-title lato">Lecture</h3>
+        <p class="gray lato main-li-p">Listen to the speakers from variouse contries about the renewable energy sources</p>
+      </li>
+      <li class="main-li">
+        <img src="src/icons/exhibition.webp" alt="Exhibition Icon" class="icon">
+        <h3 class="orange main-li-title lato">RE Exhibition</h3>
+        <p class="gray lato main-li-p">Appreciate various creations applying RE license, organized from University of Toronto</p>
+      </li>
+      <li class="main-li">
+        <img src="src/icons/forum.webp" alt="Forum Icon" class="icon">
+        <h3 class="orange main-li-title lato">Forum</h3>
+        <p class="gray lato main-li-p">Have the time to share your thoughts and opinions with experts for each topic.</p>
+      </li>
+      <li class="main-li">
+        <img src="src/icons/workshop.webp" alt="Workshop Icon" class="icon">
+        <h3 class="orange main-li-title lato">Workshop</h3>
+        <p class="gray lato main-li-p">Try creating your own work using open source license rather than just watching at it.</p>
+      </li>
+      <li class="main-li">
+        <img src="src/icons/ignite.webp" alt="Ignite Icon" class="icon">
+        <h3 class="orange main-li-title lato">RE Ignite</h3>
+        <p class="gray lato main-li-p">Get opportunities to network with RE affiliates around the world, also after the summit.</p>
+      </li>
+    </ul>
```

Comment:
This section looks great and has been rendered quite well. However, the unordered list is not the best choice for us to render the main program cards. 

- [ ] We could improve this code block by trying to render them using `<article>` tags. What do you think about the following setup 👇🏻?

```
<div>
  <article>
    <img />
    <h3> --- </h3>
    <p> --- </p>
  </article>
  <article>
    -- content goes here --
  </article>
  <article>
    -- content goes here --
  </article>
  <article>
    -- content goes here --
  </article>
  <article>
    -- content goes here --
  </article>
</div>
```

Reply:
Hi @Cathella! 👋 

That's perfect, Thanks! 😄